2022/23xxx/CVE-2022-2385.json

```diff
@@ -0,0 +1,109 @@
+{
```

Comment:
Expected location (2022/2xxx/CVE-2022-2385.json) of file does not match actual location (2022/23xxx/CVE-2022-2385.json)